articles/quickstart/native/ios-swift/09-sign-in-with-apple.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Sign In With Apple
+description: This tutorial will show you how to add a Sign In With Apple button and receive tokens from Auth0
```

Comment:
> and receive tokens from Auth0

that's not really a thing. you sign in and you optionally can get an access token to call an API. I would find a better way to describe this.